src/l10n/fr.js

```diff
@@ -0,0 +1,357 @@
+            'Do you want to restore the revision?': 'Êtes vous sur de vouloir restaurer une version antérieure ?',
+            'Shared with me'                      : 'Partagé avec moi',
+            'Shared by me'                        : 'Partagé par moi',
+            'Statistics'                          : 'Statistiques',
+            'Created on'                          : 'Crée sur',
+            'Last updated'                        : 'Dernière mise à jour',
+            '{count} revisions'                   : '{count} révisions',
+            'Shares'                              : 'Partages',
+            '{count} shares'                      : '{count} partages',
+            'More'                                : 'Plus',
+            'Backup and Restore'                  : 'Sauvegarde et Restauration',
+            'Browser Extension'                   : 'Extension navigateur',
+            'Handbook'                            : 'Manuel',
+            'Search user'                         : 'Chercher un utilisateur',
+            'Set expiration date'                 : 'Définir une date d’expiration',
+            'Expires {date}'                      : 'Expire le {date}',
+            'Choose expiration date'              : 'Choisissez une date d’expiration',
+            'Please choose a date in the future'  : 'La date doit être ultérieur à aujourd’hui',
+            'Invalid date'                        : 'Date non valide',
+            'The user {uid} does not exist'       : 'L’utilisateur {uid} n’existe pas',
+            'Unable to share password: {message}' : 'Impossible de partager le mot de passe {message}',
+            'Toggle write permissions'            : 'Changer le droit d’écriture',
+            'Toggle share permissions'            : 'Changer le droit de lecture',
+            'Stop sharing'                        : 'Arrêter le partage',
+            'Date'                                : 'Date',
+            'Default Cache (0 files, 0 B)'        : 'Cache par défaut (0 fichiers, 0 B)',
+            'Avatars Cache (0 files, 0 B)'        : 'Avatars (0 fichiers, 0 B)',
+            'Favicon Cache (0 files, 0 B)'        : 'Favicon (0 fichiers, 0 B)',
+            'Preview Cache (0 files, 0 B)'        : 'Aperçus (0 fichiers, 0 B)',
+            'Passwords Cache (0 files, 0 B)'      : 'Mots de passe (0 fichiers, 0 B)',
+            'Backup or export'                    : 'Sauvegarde/Export',
+            'Choose Format'                       : 'Choisissez un format',
+            'Please choose'                       : 'Merci de choisir',
+            'Database Backup'                     : 'Sauvegarde de la base de donnée',
+            'Predefined CSV'                      : 'Ficher CSV pré-défini',
+            'Open Office Calc'                    : 'Open/Libre Office Calc',
+            'Backup password'                     : 'Restaurer L’application',
```

Comment:
I think this is a mistake. This is to encrypt the export with a password.
Translation should be `Mot de passe de la sauvegarde`